eventsources/sources/kafka/start.go

```diff
@@ -270,6 +270,30 @@ func getSaramaConfig(kafkaEventSource *v1alpha1.KafkaEventSource, log *zap.Sugar
 		config.Version = version
 	}
 
+	if kafkaEventSource.SASL != nil {
+		config.Net.SASL.Enable = true
+
+		if kafkaEventSource.SASL.Mechanism == "" {
```

Comment:
A function `func (s SASLConfig) GetMechanism()` would be helpful to save `if..else` everywhere.

Reply:
Will refactor.

Reply:
Refactored. By default, will set to "PLAINTEXT" unless otherwise specified.